products/urls.py

```diff
@@ -1,4 +1,7 @@
 from django.urls import path
 
+from products.views import ProductListView
+
 urlpatterns = [
+    path('/list', ProductListView.as_view()),
```

Comment:
```suggestion
    path('', ProductListView.as_view()),
```
`cookit/urls.py` 에서 path를 products로 해주었기 때문에 해당 urls.py에서는 /list와 같이 작성하지 않아도 충분히 RESTful하게 상품 리스트 API에 대한 의미를 나타낼 수 있을 것 같습니다!

```shell
http -v GET 127.0.0.1:8000/products
```

Reply:
네 바로 수정했습니다~!